libraries/standard/serializer/src/cbor/iot_serializer_tinycbor_decoder.c

```diff
@@ -78,6 +103,22 @@ typedef struct _cborValueWrapper
 
 /*-----------------------------------------------------------*/
 
+static size_t calculateSizeOfCborObject( CborValue * pValue )
+{
+    IotSerializer_Assert( pValue != NULL );
+    CborValue nextValue;
+
+    /* Copy the contents of the current CBOR object so that we can advance to the next object. */
+    memcpy( &nextValue, pValue, sizeof( CborValue ) );
```

Comment:
It is same as below, right?
```
nextValue = *pValue;
```

Reply:
I'm not sure whether C will allow **assignment-copy** of `CborValue` type in a compiler-agnostic way...

Reply:
Not have a reference to C standard. But I am pretty sure that is the case. You are safe to use it. (This is not  a PR-blocking change.)

Reply:
Okay, have made the change to using assignment copy